aten/src/ATen/core/context_base.h

```diff
@@ -44,6 +45,10 @@ class CAFFE2_API BaseContext {
     SwitchToDevice(0);
   }
 
+  virtual Stream GetStream(int /*stream_id*/) {
+    return Stream{Stream::DEFAULT, Device(DeviceType::CPU, -1)};
+  }
```

Comment:
I don't think you should default this method. Fine for testing but we should follow `SwitchToDevice`.

Reply:
I'll make it a pure virtual function.